components/tabs/tabs.js

```diff
@@ -798,6 +806,14 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 				selectedTab.selected = true;
 				selectedTab.tabIndex = 0;
 			}
+
+			this._tabs.forEach((tab) => {
+				const panel = this._getPanel(tab.id);
+				if (!panel) return;
+
+				if (!panel.id) panel.id = getUniqueId();
```

Comment:
Would it be better to have the panel's constructor initialize with `id`, letting the consumer provided `id` override if provided? Having `d2l-tabs` set this id doesn't seem right, since it should really be a product of the consumer's `render` method.

Reply:
Yup I like that idea better.